### ISSUE TITLE
ES6 imports of es6 modules

### ISSUE THREAD

**Issue by hyb175** [DrkSephy/es6-cheatsheet#54]
@DrkSephy @richardbutler 
I think I picked the wrong example (React) in my earlier PR because React is exported using commonjs.

If the module is exported in es6 syntax, then this will not work.

``` javascript
const testA = 'what is it';
const testB = 'okay this works';

export default { testA, testB };
```

``` javascript
import Test, { testA, testB } from './test_export';
console.log('Test is ----> ', Test);
console.log('testA is ----> ', testA);
console.log('testB is ----> ', testB);
```

The result of the console log is

> Test is ---->  { testA: 'what is it', testB: 'okay this works' }
> testA is ---->  undefined
> testB is ---->  undefined 

I would propose to get rid of the react example altogether. I apologize for providing the wrong example in the first place.

**Comment by DrkSephy** [comment]
Would you mind sending me a pull request to replace the example with a more appropriate one?

**Comment by hyb175** [comment]
Okay. I assume you want a similar example, but I am not sure if the example makes sense. Maybe it is better to have a es6 style class with exports on both default and named export and then do the import from both default and named import

**Comment by DrkSephy** [comment]
Closed by #69. Thanks @hyb175!
